src/client/src/orangehrmClaimPlugin/pages/myClaims/MyClaims.vue

```diff
@@ -153,7 +153,8 @@ export default {
     const serializedFilters = computed(() => {
       return {
         referenceId:
-          typeof filters.value.referenceId === 'object'
+          typeof filters.value.referenceId === 'object' &&
+          filters.value.referenceId !== null
             ? filters.value.referenceId.id
             : typeof filters.value.referenceId === 'string'
             ? filters.value.referenceId
```

Comment:
@ManulMax check this
cc @Super-Chama

Reply:
Checked I think need to change ReferenceIdAutocomplete. Need a support from @Super-Chama.

Reply:
** In **ClaimRequestSearchFilterParams**, only added referenceId as allowed sort field. So as like other places we cannot search by id when he select the autocomplete result. That's why he had to get referenceId here in every time.

Reply:
is this ok ?
typeof filters.value.referenceId === 'object' &&
          filters.value.referenceId?.id !== undefined